src/index.js

```diff
@@ -1,13 +1,16 @@
 const set = require('lodash.set')
 const get = require('lodash.get')
 const dfn = require('@netlify/open-api')
-const { methods, generateMethod } = require('./open-api')
+const { addMethods } = require('./methods')
+const methods = require('./open-api/shape-swagger')
```

Comment:
Is `methods` returning the same shape?

This is used by the `netlify api` CLI command https://github.com/netlify/cli/blob/master/src/commands/api.js#L5

Reply:
`methods` is unchanged.

There is a separate PR (#66) which also does not change it. I ran `diff` before and after that PR to ensure shape was the same.

Reply:
ok cool 😎

Reply:
I've opened an issue at #75.